### PR TITLE
Re-Enable up for Alpha

### PR DESCRIPTION
**Pull request by JakeCooper**
* Up enabled for Alpha Users﻿

### CODE REVIEW

File: gateway/up.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	client := &http.Client{}
+	resp, err := client.Do(httpReq)
+	if err != nil {
+		return nil, err
+	}
+	bodyBytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+	if resp.StatusCode < 200 || resp.StatusCode > 400 {
```
Comment (coffee-cup) [inline gateway/up.go:52]:
```suggestion
	if resp.StatusCode < 200 || resp.StatusCode > 400 {
```
```suggestion
	if resp.StatusCode < 200 || resp.StatusCode >= 400 {
```

File: controller/up.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	}
+	// produce gzip
+	if err := zr.Close(); err != nil {
+		return err
+	}
+	//
+	return nil
+}
+
+func (c *Controller) Up(ctx context.Context) error {
```
[the remark this answers is not in the document]
Comment (JakeCooper) [reply controller/up.go:66]: That would mean exposing an endpoint to determine if a user is or isn't an admin. Too much leakage IMO.

[the remark this answers is not in the document]
Comment (coffee-cup) [reply controller/up.go:66]: We already have a gql endpoint setup to fetch the user. We could a also return the `isAdmin` flag there. Alternatively, we could just check if there is any user config defined (admins will obviously be logged in)

File: controller/up.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/railwayapp/cli/entity"
+)
+
+var IGNORE_FILES = map[string]bool{
```
Comment (coffee-cup) [inline controller/up.go:16]: I think this is good for now, but I still think that in the future we should use something like https://github.com/monochromegane/go-gitignore. Specifically to handle similar dep folders for other languages (or secret .env files).

**Review by coffee-cup — APPROVED** [review]

File: controller/up.go (hunk 3 above)
Comment (JakeCooper) [reply controller/up.go]: Done

**Pull request merged**